src/app/pages/ArticlePage/Byline/index.tsx

```diff
@@ -0,0 +1,253 @@
+import VisuallyHiddenText from '../../../legacy/psammead/psammead-visually-hidden-text/src';
+import BylineCss from './index.styles';
+import { RightChevron, LeftChevron } from '../../../components/icons';
+import {
+  getBodyCopy,
+  getBrevier,
+} from '../../../legacy/psammead/gel-foundations/src/typography';
+import {
+  getSansBold,
+  getSansRegular,
+  getSansRegularItalic,
+} from '../../../legacy/psammead/psammead-styles/src/font-styles';
+
+type Props = {
+  blocks: any;
+};
+
+const Byline = ({ blocks, children }: PropsWithChildren<Props>) => {
+  const { service, script, translations, dir } = useContext(ServiceContext);
+  const isRtl = dir === 'rtl';
+
+  const bylineBlocks = pathOr([], [0, 'model', 'blocks'], blocks);
+
+  const authorBlock = bylineBlocks.find((block: any) => block.type === 'name');
+  const jobRoleBlock = bylineBlocks.find((block: any) => block.type === 'role');
+  const twitterBlock = bylineBlocks.find((block: any) => block.type === 'link');
+  const locationBlock = bylineBlocks.find(
+    (block: any) => block.type === 'location',
+  );
+  const imageBlock = bylineBlocks.find((block: any) => block.type === 'image');
+
+  const author = pathOr(
+    '',
+    ['model', 'blocks', 0, 'model', 'blocks', 0, 'model', 'text'],
+    authorBlock,
+  );
+  const jobRole = pathOr(
+    '',
+    ['model', 'blocks', 0, 'model', 'blocks', 0, 'model', 'text'],
+    jobRoleBlock,
+  );
+  const twitterText = pathOr(
+    '',
+    ['model', 'blocks', 0, 'model', 'blocks', 0, 'model', 'text'],
+    twitterBlock,
+  );
+  const twitterLink = pathOr(
+    '',
+    [
+      'model',
+      'blocks',
+      0,
+      'model',
+      'blocks',
+      0,
+      'model',
+      'blocks',
+      0,
+      'model',
+      'locator',
+    ],
+    twitterBlock,
+  );
+  const location = pathOr(
+    '',
+    ['model', 'blocks', 0, 'model', 'blocks', 0, 'model', 'text'],
+    locationBlock,
+  );
+  const image = pathOr(
+    '',
+    ['model', 'blocks', 0, 'model', 'locator'],
+    imageBlock,
+  );
+
+  const contributorBlock = pathOr([], [0], blocks);
+  const authorTopicUrl = pathOr('', ['model', 'topicUrl'], contributorBlock);
+
+  if (!(author && jobRole)) return null;
+
+  const authorTranslated = pathOr('Author', ['byline', 'author'], translations);
+  const articleInformationTranslated = pathOr(
+    'Article information',
+    ['byline', 'articleInformation'],
+    translations,
+  );
+  const jobRoleTranslated = pathOr('Role', ['byline', 'role'], translations);
+
+  const publishedTranslated = pathOr(
+    'Published',
+    ['byline', 'published'],
+    translations,
+  );
+  const reportingFromTranslated = pathOr(
+    'Reporting from',
+    ['byline', 'reportingFrom'],
+    translations,
+  );
+
+  return (
+    <section
+      css={BylineCss.bylineSection}
+      role="region"
+      aria-labelledby="article-byline"
+    >
+      <VisuallyHiddenText id="article-byline" aria-hidden>
+        {articleInformationTranslated}
```

Comment:
Could we please output a strong tag here instead of a span if we are using this method of labelling

Reply:
Done